sorc/ncep_post.fd/CALHEL3.f

```diff
@@ -458,6 +458,10 @@ SUBROUTINE CALHEL3(LLOW,LUPP,UST,VST,HELI)
                 DV1 = VH(I,J,L+1)-VH(I,J,L)
                 DV2 = VH(I,J,L)-VH(I,J,L-1)
               IF( L >= LUPP(I,J) .AND. L <= LLOW(I,J) ) THEN
+               IF( VH(I,J,L)  <spval.and.UH(I,J,L)  <spval.and.         &
+                   VH(I,J,L+1)<spval.and.UH(I,J,L+1)<spval.and.         &
+                   VH(I,J,L-1)<spval.and.UH(I,J,L-1)<spval.and.         &
+                   VST(I,J)   <spval)                                   &
```

Comment:
please add ust in line 464.
@JesseMeng-NOAA

Reply:
will do